### PR TITLE
Revert "go generate -run=mockgen ./..."

### PR DESCRIPTION
**Pull request by gdbelvin**
And turn off travis code generation check. 

This is necessary because gomock has been updated upstream in an incompatible way. 
We need to wait a bit before we run `go generate mockgen` again. As long as no substantial interfaces change in that time we should be fine. 

The alternative is to pin mockgen at a specific revision, but `go get` does not support this and would require a bit of a hack. 

Reverts google/trillian#1378

### CODE REVIEW

**Comment by RJPercival** [comment]
Would vendoring gomock help at all here? If not, you'll have to exclude mocks from the check Travis does here:
https://github.com/google/trillian/blob/5c01ad8a7d14741f2eb0d0d56163cac489f7263b/.travis.yml#L96

**Comment by daviddrysdale** [comment]
There's a weird empty commit that talks about reverting #1377 too -- can that commit be dropped?

**Comment by RJPercival** [comment]
Using the --no-generate flag will disable useful checks, like whether the API auto-generated documentation is up-to-date. Could you just exclude mocks from the check instead?

**Comment by gdbelvin** [comment]
Removed the spurious revert commit and isolated the presubmit check to just ignore gomock.

File: scripts/presubmit.sh [hunk 1]
```diff
@@ -156,7 +156,7 @@ main() {
 
     echo 'running go generate'
     go generate -run="protoc" ./...
-    go generate -run="mockgen" ./...
+    # go generate -run="mockgen" ./... # TODO(gbelvin) renable.
```
Comment (daviddrysdale) [inline scripts/presubmit.sh]: super-nit: "re-enable" or "reenable"

**Pull request merged**